rmw_zenoh_cpp/src/detail/liveliness_utils.cpp

```diff
@@ -446,6 +446,24 @@ std::string mangle_name(const std::string & input)
   return output;
 }
 
+///=============================================================================
+char * mangle_name(const char * input)
```

Comment:
Unnecessary code duplication.

Reply:
I try to resolve this. Please review it.